apps/coscrad-frontend/src/utils/generic-components/presenters/detail-views/resource-preview-icon.tsx

```diff
@@ -68,5 +68,10 @@ export const ResourcePreviewIconFactory = ({
         throw new Error(`Failed to build an icon for resource type: ${resourceType}`);
     }
 
-    return <Box sx={{ fontSize: iconSize, maxHeight: iconSize, color: color }}>{lookupResult}</Box>;
+    return (
+        <Box sx={{ fontSize: iconSize, maxHeight: iconSize, color: color }}>
+            {/* TODO: capitalize resource type */}
```

Comment:
text-transform: "capitalize" ?

Reply:
Good call, but maybe I'll wait for now for the official COSCRAD utility